axiom/nr_php_packages.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef nr_php_packages_HDR
+#define nr_php_packages_HDR
```

Comment:
The paradigm for header `#ifdef`'s are all caps:
```suggestion
#ifndef NR_PHP_PACKAGES_HDR
#define NR_PHP_PACKAGES_HDR
```